rust/src/lib/ifaces/bond.rs

```diff
@@ -104,8 +105,17 @@ impl BondInterface {
         }
     }
 
+    fn sort_ports_config(&mut self) {
+        if let Some(ref mut bond_conf) = self.bond {
+            if let Some(ref mut port_conf) = &mut bond_conf.ports_config {
+                port_conf.sort_unstable_by_key(|p| p.name.clone())
```

Comment:
Just a rust quiz: could you explain why we need `clone()` here?

Reply:
It relates to Rust's ownership, which prevents moving the element in a collection during sorting unless the elements are cloned ?